motoman_driver/MotoPlus/SimpleMessage.h

```diff
@@ -133,6 +134,9 @@ typedef enum
 	ROS_CMD_CHECK_MOTION_READY = 200101,
 	ROS_CMD_CHECK_QUEUE_CNT = 200102,
 	ROS_CMD_STOP_MOTION = 200111,
+	ROS_CMD_START_SERVOS = 200112, // starts the servo motors

```

Comment:
@ted-miller, correct me if I am wrong, but I believe these are "sub-command" types that are under a general Motoman SimpleMessage command ID.  Simple message IDs and ranges are specified [here](https://github.com/ros-industrial/rep/blob/master/rep-I0004.rst).  There isn't any overlap between these sub command types and anything envisioned in the simple message spec.  I believe the number decision was Motoman's.